src/nxdoc/nuxeo-add-ons/nuxeo-mail-amazon-ses.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Nuxeo Mail Amazon SES
+description: The Nuxeo Mail Amazon SES addon allows you to send emails via AWS SES. To do so, it adds a Nuxeo MailSender implementation specific to Amazon SES which overrides the default SMTPMailSender and leverages the Nuxeo MailService.
+tree_item_index: 350
+---
+
+The [Nuxeo Mail Amazon SES](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-mail-amazon-ses) addon allows you to send emails via [AWS SES](https://docs.aws.amazon.com/ses). To do so, it adds a Nuxeo `MailSender` implementation specific to Amazon SES which overrides the `"default"` `SMTPMailSender` and leverages the Nuxeo `MailService`.
+
+## Before You Start
+
+You should be familiar with Amazon AWS and in possession of your credentials and region.
+You should also be familiar with [SES reputation](https://docs.aws.amazon.com/ses/latest/dg/monitor-sender-reputation.html) and [identity verification](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html).
```

Comment:
I couldn't get versionned javadoc. It seems they only provide latest. Otherwise this would be `1.12.261` as of today